mcc/odin/odin/snapshot_list.go

```diff
@@ -0,0 +1,18 @@
+package odin
+
+import (
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
+)
+
+// ListSnapshots return a list of all DBSnapshots.
+func ListSnapshots(
+	instanceName string,
+	svc rdsiface.RDSAPI,
+) (
+	result []*rds.DBSnapshot,
```

Comment:
not a correction on @dani comment, just to point that in this case, the return values are named, so these become already declared variables, allowing to use `return` without specifying the variable names.